addons/default/visiosoft/profile-module/resources/views/profile/index.twig

```diff
@@ -1,5 +1,6 @@
 {% block styles %}
     {{ asset_style("visiosoft.module.profile::assets/css/profile-nav.scss") }}
+    {{ asset_style("visiosoft.module.profile::assets/css/profile.scss") }}
```

Comment:
why did you add the profile.scss here?

Reply:
https://github.com/openclassify/openclassify/pull/686/commits/d40f7dd2826618b9b098b53e3f00cc7f6ff33011 @diashalabi